libs/permission-checker/src/Role.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker;
+
+enum Role: string
+{
+    case NONE = 'none';
+    case ADMIN = 'admin';
+    case GUEST = 'guest';
+    case READ_ONLY = 'readOnly';
+    case SHARE = 'share';
+    case DEVELOPER = 'developer';
+    case REVIEWER = 'reviewer';
+    case PRODUCTION_MANAGER = 'productionManager';
```

Comment:
nalinkuj sem prosímtě https://github.com/keboola/connection/blob/d9e2a3cd927d133449bdd66b44786d5ba80f96c3/legacy-app/library/Keboola/Validate/ProjectRole.php ať víme, kde se to bere

Reply:
doplneno